test/Npgsql.Tests/YBTestUtils.cs

```diff
@@ -66,11 +67,26 @@ protected static async Task VerifyOn(string server, int ExpectedCount)
             var responseBody = await response.Content.ReadAsStringAsync();
             var count = responseBody.Split("client backend");
             Console.WriteLine(server + ":" + (count.Length - 1));
-            Assert.AreEqual(ExpectedCount, count.Length - 1);
+            // Assert.AreEqual(ExpectedCount, count.Length - 1);
```

Comment:
Removed unnecessary line breaks and commented out lines

Reply:
Done